SpatialGDK/Documentation/toc.md

```diff
@@ -6,7 +6,7 @@
     - [2: Get and build the GDK’s Unreal Engine Fork]({{urlRoot}}/content/get-started/build-unreal-fork)
     - [3: Follow the Multiserver Shooter tutorial]({{urlRoot}}/content/get-started/tutorial)
     - Next steps - either:
-        - [Set up and run the Starter Project]({{urlRoot}}/content/get-started/gdk-and-starter-project)
+        - [Set up the SpatialOS GDK template]({{urlRoot}}/content/get-started/gdk-template)
```

Comment:
This title is not consistent with the title in the page header and the in-page text. We should settle on a consistent title, IMHumbleO and consistent capitalisation or it's not clear we are talking about the same project.